config/karma-docs.conf.js

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Can you add a top level comment explaining what this karma configuration is for?